pkg/cache/cache_test.go

```diff
@@ -985,25 +1044,37 @@ func TestCacheClusterQueueOperations(t *testing.T) {
 			if err := tc.operation(cache); err != nil {
 				t.Errorf("Unexpected error during test operation: %s", err)
 			}
-			if diff := cmp.Diff(tc.wantClusterQueues, cache.clusterQueues,
-				cmpopts.IgnoreFields(clusterQueue{}, "Cohort", "ResourceGroups"),
+			if diff := cmp.Diff(tc.wantClusterQueues, cache.hm.ClusterQueues,
+				cmpopts.IgnoreFields(clusterQueue{}, "ResourceGroups"),
 				cmpopts.IgnoreFields(workload.Info{}, "Obj", "LastAssignment"),
-				cmpopts.IgnoreUnexported(clusterQueue{}),
+				cmpopts.IgnoreUnexported(clusterQueue{}, hierarchy.WiredClusterQueue[*clusterQueue, *cohort]{}),
 				cmpopts.EquateEmpty()); diff != "" {
 				t.Errorf("Unexpected clusterQueues (-want,+got):\n%s", diff)
 			}
 
-			gotCohorts := make(map[string]sets.Set[string], len(cache.cohorts))
-			for name, cohort := range cache.cohorts {
+			gotCohorts := make(map[string]sets.Set[string], len(cache.hm.Cohorts))
+			for name, cohort := range cache.hm.Cohorts {
 				gotCohort := sets.New[string]()
-				for cq := range cohort.Members {
+				for _, cq := range cohort.Members() {
 					gotCohort.Insert(cq.Name)
 				}
 				gotCohorts[name] = gotCohort
 			}
 			if diff := cmp.Diff(tc.wantCohorts, gotCohorts, cmpopts.EquateEmpty()); diff != "" {
 				t.Errorf("Unexpected cohorts (-want,+got):\n%s", diff)
 			}
+			for _, cq := range cache.hm.ClusterQueues {
```

Comment:
I think these two-way checks are a bit hard to read. Would it be cleaner to have `wantCqEdges` and assert similarly as suggested in the comment to "verify cq<->cohort edge count"?

Reply:
Discussed offline with @mimowo: decided to delete these tests, as it is hierarchy_manager's job to maintain consistency of edges, so these tests are duplicates. We are already verify CQs in Cohorts a few lines above